Jenkinsfile

```diff
@@ -20,28 +20,88 @@ pipeline {
+                                    }
                                 }
                             }
                         }
                     }
                 )
             }
         }
+        stage('Static Analysis') {
+            steps {
+                parallel(
+                        owasp: {
+                            node('linux-large') {
+                                checkout scm
+                                withMaven(maven: 'M3', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings') {
+                                    sh 'mvn install -q -B -Powasp -DskipTests=true -Dpmd.skip=true -Dfindbugs.skip=true -Djacoco.skip=true -Dcheckstyle.skip=true -pl !libs/owasp-diff-runner,!distribution/docs'
+                                }
+                            }
+                        },
+                        sonarqube: {
+                            node('linux-large') {
+                                checkout scm
+                                withMaven(maven: 'M3', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings') {
+                                    withCredentials([string(credentialsId: 'sonarqube-token', variable: 'SONAR_TOKEN')]) {
+                                        sh 'mvn -q -B -Dfindbugs.skip=true -Dcheckstyle.skip=true org.jacoco:jacoco-maven-plugin:prepare-agent install sonar:sonar -Dsonar.host.url=https://sonarqube.com -Dsonar.login=$SONAR_TOKEN  -Dsonar.organization=codice -Dsonar.projectKey=ddf -pl !platform/solr/platform-solr-server-standalone,!distribution/docs,!distribution/test/itests/test-itests-ddf'
+                                    }
+                                }
+                            }
+                        },
+                        coverity: {
+                            node('linux-medium') {
+                                checkout scm
+                                withMaven(maven: 'M3', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings') {
+                                    withCredentials([string(credentialsId: 'ddf-coverity-token', variable: 'COVERITY_TOKEN')]) {
+                                        withEnv(["PATH=${tool 'coverity-linux'}/bin:${env.PATH}"]) {
+                                            sh '''cov-build --dir cov-int mvn -B -DskipTests=true -Dfindbugs.skip=true -Dcheckstyle.skip=true -Djacoco.skip=true install -pl !distribution/docs --settings $MAVEN_SETTINGS
+                                              tar czvf ddf.tgz cov-int
+                                              curl --form token=$COVERITY_TOKEN --form email=cmp-security-team@connexta.com --form file=@ddf.tgz --form version="master" --form description="Description: from CI" https://scan.coverity.com/builds?project=codice%2Fddf'''
```

Comment:
Should updated the description here? I think it was fine but was a bit generic for coverity pushes.